zulipterminal/helper.py

```diff
@@ -751,9 +753,7 @@ def process_media(controller: Any, media_link: str) -> None:
             for chunk in r.iter_content(chunk_size=8192):
                 if chunk:  # Filter out keep-alive new chunks.
                     f.write(chunk)
-                    controller.view.set_footer_text(
-                        [" Downloading ", ("bold", media_name)]
-                    )
+                    callback([" Downloading ", ("bold", media_name)])
```

Comment:
This is not coupled to the controller etc, but you need to know that it's an urwid styled-text being passed in. You could specify the parameter in the calling function, and have this just be `callback()`. A different name would help too - see elsewhere.

Reply:
Right. I just passed in the params to show the media name as well. But I think showing just `Downloading your media ...` is good, since I'm prompting the user the filename while opening it in any case :+1: